src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/World.kt

```diff
@@ -113,7 +116,13 @@ object World {
     fun getChunk(x: Int, y: Int, z: Int) = Chunk(getWorld()!!.getWorldChunk(MCBlockPos(x, y, z)))
 
     @JvmStatic
-    fun getAllEntities() = getWorld()?.entities?.map(::Entity) ?: listOf()
+    fun getAllEntities() = getWorld()?.entities?.map {
+        when (it) {
+            is PlayerEntity -> PlayerMP(it)
+            is MCLivingEntity -> LivingEntity(it)
+            else -> Entity(it)
+        }
+    } ?: listOf()
```

Comment:
This is a great change, it might be nice to add an `Entity.from` or something so modules could use this functionality also. Maybe `fromMC()`, since I've been trying to expose all MC-types as `.toMC()`